scripts/technical-debt-metrics.sh

```diff
@@ -84,6 +84,8 @@ titlesPathsAndRegexes=(
   "disabled simpNF lints"          "*"      "nolint simpNF"
   "erw"                            "*"      "erw \["
   "maxHeartBeats modifications"    ":^MathlibTest" "^ *set_option .*maxHeartbeats.* [0-9][0-9]*"
+  "CommRing (Fin n) instances"     "*"      "^open Fin.CommRing "
```

Comment:
I am not entirely clear about how this works.  Why is there a space at the end?  And what about opening `Fin.CommRing` and other namespaces before/after?

Reply:
[This branch](https://github.com/leanprover-community/mathlib4/tree/MR-lint-finnatcast) starts adding a linter, but doesn't quite work yet.

Reply:
With the space, though, this would not match a line consisting exactly of `open Fin.CommRing`, right?

Reply:
To match a word separator, you can use `\b`, so something like `open Fin.CommRing\b`.

I am also not sure of whether we should be looking at `open Fin.CommRing in` as exceptions or not.

Reply:
Rather than worrying too much, let's merge #25531 which gets rid of many of them!